<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<arg name="frame_id"                         default="map"/>
	<arg name="max_range"                        default="10.0"/>
	<arg name="min_range"                        default="1.0"/>
	<arg name="resolution"                       default="1.0"/>


	<node pkg="temporal_octomap" type="temporal_octomap_node" name="temporal_octomap"> 
		
		<remap from="cloud_in" 					to="/PointCloud" />

		<param name="frame_id"                   value="$(arg frame_id)"/>
        <param name="max_range"                  value="$(arg max_range)"/>
        <param name="min_range"               	 value="$(arg min_range)"/>
        <param name="resolution"                 value="$(arg resolution)"/>
	</node>
</launch>